template_content/pyproject.toml.jinja

```diff
@@ -30,6 +30,9 @@ pytest-cov = "*"
 {% if use_python_pip_audit -%}
 pip-audit = "*"
 {% endif %}
+{% if use_pre_commit -%}
+pre-commit = "*"
+{% endif -%}
```

Comment:
Minor thing, but to keep the whitespace consistent all the endifs in this section should be `{% endif -%}` except the last one should omit the `-` e.g. `{% endif %}`